src/meta/src/rpc/ddl_controller.rs

```diff
@@ -174,6 +246,12 @@ where
                     ctrl.drop_view(view_id, drop_mode).await
                 }
                 DdlCommand::CreateStreamingJob(stream_job, fragment_graph) => {
+                    let _x = ctrl
```

Comment:
Rename to emphasize that its life cycle is used to release the permit

```suggestion
                    let _permit = ctrl
```

Reply:
I suggest moving this line into `ctrl.create` to keep this function still a simple dispatching.